docs/maps/search.asciidoc

```diff
@@ -0,0 +1,37 @@
+[role="xpack"]
+[[maps-search]]
+== Searching Your Data
+
+The **Maps** application embeds the query bar with autocomplete for real-time ad hoc search.
+All layers with {es} sources are updated when you submit a search request.
+
+image::maps/images/global_search_bar.png[]
+
+
+[float]
+=== Layer with {es} source
+
+You can create a layer with {es} source from the following:
+
+- <<vector-layer>> with *Documents* source.
+
+- <<vector-layer>> with *Grid aggregation* source.
+
+- <<heatmap-layer>> with *Grid aggregation* source.
+
+- <<vector-layer>> with <<terms-join>>.
+
+
+[float]
+=== Searching across multiple indexes
```

Comment:
```suggestion
=== Searching across multiple indices
```

Reply:
I used `indexes` to be consistent with other places in Elasticsearch docs, such as https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-index-field.html. Is there an effort to migrate `indexes` to `indices`? Which is preferred?

Reply:
The Elasticsearch writing style guide says to use "indices."